### ISSUE TITLE
Can I edit the local colors？

### ISSUE THREAD

**Issue by BCL123456-BAL** [yuehaowang/RecolorNeRF#5]
Thank you for your good work！
I would like to recolor a local area of the target, but from what the paper says, it seems that this would change the overall color.

**Comment by yuehaowang** [comment]
Yes, the current method is a pure palette-based color editing approach.
To enable color editing on a local areas, a possible solution is to incorporate our proposed method into some other backbones with semantics, e.g., distilled feature fields.
